pkg/loop/internal/pb/reporting_ocr3.proto

```diff
@@ -0,0 +1,149 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb";
```

Comment:
What if we moved these to a subpackage, and dropped the `OCR3` prefixes?
```suggestion
option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ocr3";
```

Reply:
Another option:
```suggestion
option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/ocr3/pb";
```
:shrug:

Reply:
i choose internal/pb/mercury here. let's keep it consistent https://github.com/smartcontractkit/chainlink-common/pull/304/files